src/script/calling/CallingRepository.ts

```diff
@@ -797,7 +797,7 @@ export class CallingRepository {
       if (newState === VIDEO_STATE.STOPPED) {
         selfParticipant.releaseVideoStream(true);
       } else {
-        this.warmupMediaStreams(call, false, true);
+        void this.warmupMediaStreams(call, false, true);
```

Comment:
<img width="1048" alt="image" src="https://user-images.githubusercontent.com/63250054/236863151-dd70e772-de2c-4aa3-a811-91956082034b.png">
without that we get this warning but there is no need to catch anything on this, all code inside warmupMediaStreams is in a try catch and has it's possible error handled

Reply:
this feels slightly incorrect to me, as though we are missing the correct async/awaits on things and this will bite us later when we cant figure out why something isn't awaiting that should be

Reply:
id prefer to leave the warning. my sense is that it is a *correct* warning for the callingRepo and we should be doing something with all of these.

Reply:
removed void operators for now: https://github.com/wireapp/wire-webapp/pull/14600/commits/877a88c05f673c2c891cb49b72b174b77d1c3cd8